src/v/cloud_storage/remote.cc

```diff
@@ -1580,7 +1580,6 @@ void auth_refresh_bg_op::do_start_auth_refresh_op(
                   return cloud_roles::aws_region_name{cfg.region};
               },
               [](cloud_storage_clients::abs_configuration const&) {
-                  vassert(false, "Attempt to create refresh creds for ABS");
                   return cloud_roles::aws_region_name{};
```

Comment:
do we have a region name now?

Reply:
> do we have a region name now?

> vassert(false, "Attempt to create refresh creds for ABS");

This assert is removed, but we dont' fill in the aws_region_name. Do we want to? Why do we return something called aws_region_name for an abs_configuration?

Reply:
refresh_credential::impl::impl has a common implementation for s3/gcp and abs, and takes an aws_region_name. 

only s3 needs it to compose the requests, abs ignores it.

it was easier to not touch this part of the implementation. 

Adding a comment on this part